packages/deployment/scripts/governance/roles-gov-v2.ts

```diff
@@ -48,16 +48,27 @@ export async function rolesGovV2(useBummerConfig = false) {
   console.log('[PROTOCOL ACCESS MANAGER] - Setting up governance...')
 
   // Handle timelock governor role
-  const hasGovernorRole = await protocolAccessManager.read.hasRole([
+  const hasTimelockGovernorRole = await protocolAccessManager.read.hasRole([
     GOVERNOR_ROLE,
     timelock.address,
   ])
-  if (!hasGovernorRole) {
+  if (!hasTimelockGovernorRole) {
     console.log('[PROTOCOL ACCESS MANAGER] - Granting governor role to timelock...')
     const hash = await protocolAccessManager.write.grantGovernorRole([timelock.address])
     await publicClient.waitForTransactionReceipt({ hash })
   }
 
+  // Handle timelock governor role
+  const hasGovGovernorRole = await protocolAccessManager.read.hasRole([
```

Comment:
The name is a bit confusing, particularly when it talks about the time lock governor role in the comment. What is GovGovernor?